### PR TITLE
CAS-187 - add helper text to Community Links

### PR DESCRIPTION
**Pull request by germanurrus**
Screen Shot 2022-07-12 at 13 06 54

### CODE REVIEW

File: frontend/packages/client/src/components/Community/CommunityEditorLinks.js [hunk 1]
```diff
@@ -34,6 +40,8 @@ export const CommunityLinksForm = ({
   wrapperMargin = 'mb-6',
   wrapperMarginMobile = 'mb-4',
 }) => {
+  const { register, formState } = useFormContext();
+
```
Comment (germanurrus) [inline frontend/packages/client/src/components/Community/CommunityEditorLinks.js:44]: using context to plug react-hook

File: frontend/packages/client/src/components/Community/hooks/useLinkValidator.js [hunk 2]
```diff
@@ -16,30 +16,29 @@ export const urlPatternValidation = (url) => {
...
+    /(https:\/\/)(www\.)(discord\.(gg|io|me|li|com)|discordapp\.com\/invite)\/.+[a-zA-Z0-9]/gim.test(
       url
     )
   );
 };
 const githubValidator = (url) => {
-  return url === '' || /https?:\/\/github\.com\/(?:[^/\s]+)/gim.test(url);
+  return (
+    url === '' || /(https:\/\/)(www\.)(github\.com)\/(?:[^/\s]+)/gim.test(url)
+  );
```
Comment (germanurrus) [inline frontend/packages/client/src/components/Community/hooks/useLinkValidator.js:41]: changes done to enforce patter `https://www.` to be consistent with place holder

Comment (dbslone) [reply frontend/packages/client/src/components/Community/hooks/useLinkValidator.js:41]: The `www.` should still be optional in the regex

**Comment by germanurrus** [comment]
@dbslone updates done

**Review by dbslone — APPROVED** [review]

**Pull request merged**